xbmc/settings/Settings.h

```diff
@@ -138,14 +138,21 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static constexpr auto SETTING_MYVIDEOS_STACKVIDEOS = "myvideos.stackvideos";
   static constexpr auto SETTING_LOCALE_SUBTITLELANGUAGE = "locale.subtitlelanguage";
   static constexpr auto SETTING_SUBTITLES_PARSECAPTIONS = "subtitles.parsecaptions";
+  static constexpr auto SETTING_SUBTITLES_CAPTIONSALIGN = "subtitles.captionsalign";
```

Comment:
In the future we likely want to move subtitle setting operations to the subtitlesettings file. At least it's one of my goals